src/easyscience/Objects/variable/descriptor_array.py

```diff
@@ -0,0 +1,760 @@
+from __future__ import annotations
+
+import numbers
+import operator as op
```

Comment:
hiding "internal" name is prone to errors, especially when you start using `operator` as a string later on